nilearn/datasets/func.py

```diff
@@ -398,8 +398,8 @@ def fetch_adhd(n_subjects=None, data_dir=None, url=None, resume=True,
     Parameters
     ----------
     n_subjects: int, optional
-        The number of subjects to load. If None is given, all the
-        40 subjects are used.
+        The number of subjects to load from maximum of 40 subjects.
+        If default None, only 30 subjects will be loaded.
```

Comment:
No, that exactly what I would like the change to be about. I'd like "n_subjects=30" explicitly by default in the definition of this function.